test cases/common/160 has header use/meson.build

```diff
@@ -0,0 +1,5 @@
+project('has header use', 'c')
+
+c_compiler = meson.get_compiler('c')
+
+assert(not c_compiler.has_header('header.h', use : true), 'non-usable header is usable')
```

Comment:
This test may fail simply because it is being run from a different directory so `header.h` is not in the search path. There should be two parts in this test. First a header `header_canbeused.h` that should work and return true for `has_header` with `use : true` and then the current test, whose header name should be 'header_cannotbeused.h` or somesuch.